app/jobs/models.py

```diff
@@ -0,0 +1,91 @@
+from datetime import timedelta
+
+from django.conf import settings
+from django.db import models
+from django.urls import reverse
+from django.utils import timezone
+from django.utils.translation import ugettext_lazy as _
+
+# Create your models here.
+
+JOB_TYPE_CHOICES = (
+    ('full_time', _('Full-Time')),
+    ('part_time', _('Part-Time')),
+    ('contract', _('Contract')),
+    ('intern', _('Intern')),
+)
+
+
+def get_expiry_time():
+    return timezone.now() + timedelta(days=30)
+
+
+class Job(models.Model):
+    title = models.CharField(
+        verbose_name=_('Title'), max_length=200, null=False, blank=False
+    )
+    description = models.TextField(verbose_name=_('Description'))
+    github_profile_link = models.URLField(
+        verbose_name=_('Github Profile Link'), null=False, blank=True
+    )
+    location = models.CharField(
+        _('Location'), max_length=50, null=False, blank=True
+    )
+    job_type = models.CharField(
+        _('Job Type'), max_length=50, choices=JOB_TYPE_CHOICES, null=False,
+        blank=False
+    )
+    apply_url = models.URLField(null=False, blank=True)
+    is_active = models.BooleanField(
+        verbose_name=_('Is this job active?'), default=False,
+        null=False, blank=True
+    )
+    skills = models.CharField(
+        verbose_name=_('skill'), max_length=60, null=True, blank=True
+    )
+    expiry_date = models.DateTimeField(
+        _('Expiry Date'), null=False, blank=False, default=get_expiry_time
+    )
+    company = models.CharField(_('Company'), max_length=50, null=True, blank=True)
+    apply_email = models.EmailField(_('Contact Email for Job'), null=True, blank=True)
+    posted_at = models.DateTimeField(_('Posted At'), null=False, blank=False, default=timezone.now)
+    posted_by_gitcoin_username = models.CharField(
```

Comment:
This should be associated with the `dashboard.Profile` that created the `Job` versus charfield of username